dashboards/google-microservices/microservices-grpc-monitoring-gce.json

```diff
@@ -414,7 +696,7 @@
                 "plotType": "LINE",
                 "targetAxis": "Y1",
                 "timeSeriesQuery": {
-                  "timeSeriesQueryLanguage": "fetch gce_instance\n  | { t_0:\n      metric 'custom.googleapis.com/opencensus/grpc.io/server/started_rpcs'\n        | align delta(1m)\n        | every 1m\n      ; t_1:\n      metric 'custom.googleapis.com/opencensus/grpc.io/server/completed_rpcs'\n        | align delta(1m)\n        | every 1m\n        | group_by drop [metric.grpc_server_status],\n            [completed_rpcs_aggregate: aggregate(value.completed_rpcs)] }\n  | outer_join [0]\n  | value [concurrent_rpcs: sub(t_0.started_rpcs, t_1.completed_rpcs_aggregate)]\n  | filter (val() >= 0)\n"
+                  "prometheusQuery": "(\n  sum by(grpc_server_method, opencensus_task, project_id, instance_id, zone) (\n    increase(custom_googleapis_com:opencensus_grpc_io_server_started_rpcs{\n      monitored_resource=\"gce_instance\"\n    }[1m])\n  )\n  -\n  sum by(grpc_server_method, opencensus_task, project_id, instance_id, zone) (\n    increase(custom_googleapis_com:opencensus_grpc_io_server_completed_rpcs{\n      monitored_resource=\"gce_instance\"\n    }[1m])\n  )\n)\nand\n(\n  sum by(grpc_server_method, opencensus_task, project_id, instance_id, zone) (\n    increase(custom_googleapis_com:opencensus_grpc_io_server_started_rpcs{\n      monitored_resource=\"gce_instance\"\n    }[1m])\n  )\n  -\n  sum by(grpc_server_method, opencensus_task, project_id, instance_id, zone) (\n    increase(custom_googleapis_com:opencensus_grpc_io_server_completed_rpcs{\n      monitored_resource=\"gce_instance\"\n    }[1m])\n  ) >= 0\n)\n"
```

Comment:
Same question as above about whether you need to `and` the expression against itself